packages/ui-components/src/components/Inputs/Autocomplete.js

```diff
@@ -145,4 +148,5 @@ Autocomplete.defaultProps = {
   inputRef: null,
   name: null,
   defaultValue: null,
+  tooltip: '',
```

Comment:
Any reason not to make it null?

Reply:
Purely because it's a string type, so could be either really!